source/mir/ndslice/slice.d

```diff
@@ -525,8 +535,13 @@ pure nothrow unittest
     assert(slice.elementsCount == 5 * 6 * 7);
     static assert(is(typeof(slice) == Slice!(3, int*)));
 
-    auto duplicate = createSlice!int(slice.shape);
-    duplicate[] = slice;
+    auto dup1 = createSlice!int(slice.shape);
```

Comment:
Hmm I am not sure whether this should be part of the first public unittest. It could be overwhelming to the user.